Source/RyuInstaller/RyuInstaller.GUI/UIUserInput.cs

```diff
@@ -0,0 +1,53 @@
+﻿using MaterialSkin.Controls;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace RyuInstaller.GUI
+{
+    public partial class UIUserInput : MaterialForm
+    {
+        public static string _Title;
+        public static string _TextHint;
+        public static string _TextError;
+        public static string _TextSuccess;
+        public static int _ReturnValue;
+
+        public UIUserInput(string Title, string TextHint, string TextError, string TextSuccess, int ReturnValue)
```

Comment:
arguments names should be in camelCase
```suggestion
        public UIUserInput(string title, string textHint, string textError, string textSuccess, int returnValue)
```

but `_TextError`, `_TextSuccess` and `_ReturnValue` seems to be unused in the rest of the class.